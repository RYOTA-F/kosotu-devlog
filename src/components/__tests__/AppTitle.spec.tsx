import React from 'react'
import renderer from 'react-test-renderer'
import AppTitle from '../AppTitle'

test('AppTitle', () => {
  const component = renderer.create(<AppTitle />)
  const tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})