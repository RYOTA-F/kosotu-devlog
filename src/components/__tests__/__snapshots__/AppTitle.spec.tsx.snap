// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle 1`] = `
<h1
  className="sc-bdvvaa btYIOi"
>
  message
</h1>
`;
